### PR TITLE
samples: create BigQuery subscription

### PR DESCRIPTION
**Pull request by kamalaboulhosn**

### CODE REVIEW

**Review by anguillanneuf — CHANGES_REQUESTED** [review]
Mostly LGTM. Resource creation+deletion should be moved out of the test. 

Samples tests fail due to:

```
>       _delete_bigquery_table(dataset_id, delete_contents=True)
E       TypeError: _delete_bigquery_table() got an unexpected keyword argument 'delete_contents'
```

File: samples/snippets/subscriber_test.py [hunk 1]
```diff
@@ -545,6 +545,68 @@ def test_update_push_subscription(
...
+    ]
+
+    table = bigquery.Table(table_id, schema=schema)
+    table = client.create_table(table)
+
+
+def _delete_bigquery_table(dataset_id: str) -> None:
+    client = bigquery.Client()
+    dataset = bigquery.Dataset(f"{PROJECT_ID}.{dataset_id}")
+    client.delete_dataset(dataset)
```
[the remark this answers is not in the document]
Comment (kamalaboulhosn) [reply samples/snippets/subscriber_test.py]: Done.

**Review by anguillanneuf — APPROVED** [review]
The samples tests failures seem unrelated. @acocuzzo to investigate. #730 may be the fix.

File: samples/snippets/subscriber_test.py [hunk 2]
```diff
@@ -545,6 +548,63 @@ def test_update_push_subscription(
...
+    client.delete_dataset(dataset, delete_contents=True)
+
+
+def test_create_bigquery_subscription(
+    subscriber_client: pubsub_v1.SubscriberClient,
+    topic: str,
+    bigquery_table: str,
+    capsys: CaptureFixture[str],
+) -> None:
+    underscored_version = PY_VERSION.replace(".", "_")
```
Comment (acocuzzo) [inline samples/snippets/subscriber_test.py:581]: This unused variable is causing linter failure, please remove

**Pull request merged**